src/main/resources/creatures.json

```diff
@@ -311,7 +311,7 @@
       "tags": [
         "CORPSE"
       ],
-      "drop": [
+      "drops": [
```

Comment:
@UnknownNPC thanks to your work, this was found. Obviously we didn't use the "drop" field, and this caused a minor gameplay glitch. Tests being worth it already.